browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -85,6 +89,33 @@ void EphemeralStorageTabHelper::ReadyToCommitNavigation(
   CreateEphemeralStorageAreasForDomainAndURL(new_domain, new_url);
 }
 
+void EphemeralStorageTabHelper::ClearEphemeralLifetimeKeepalive(
+    const content::TLDEphemeralLifetimeKey& key) {
+  ClearLocalStorageKeepAlive(
+      StringToSessionStorageId(key.second, kLocalStorageSuffix));
+
+  for (auto it = keep_alive_tld_ephemeral_lifetime_list_.begin();
```

Comment:
I'd rather do `base::ranges::find()` and then erase: this is 2 times shorter and also safer, because the current loop looks as a common pitfall (without immediate `return`). If `return` is occasionally refactored/removed, we could be in trouble

Reply:
not `ranges` because we specifically only want to remove the first one we find, but I can convert it to `std::find`